src/config/network/wpa_supplicant.md

```diff
@@ -1 +1,103 @@
-# wicd
+# wpa\_supplicant
+
+The `wpa_supplicant` package is installed by default on the base system
+and includes utilities to configure wireless interfaces and, more
+specifically, to handle wireless security protocols.
+
+[wpa\_supplicant(8)](https://man.voidlinux.org/wpa_supplicant.8) is a
+daemon that manages wireless interfaces based on a
+[wpa\_supplicant.conf(5)](https://man.voidlinux.org/wpa_supplicant.conf.5)
+configuration files.
+[wpa\_passphrase(8)](https://man.voidlinux.org/wpa_passphrase.8) and
+[wpa\_cli(8)](https://man.voidlinux.org/wpa_cli.8) are frontends to
+manage the configuration files and the running service.
+
+## Configuring wpa\_supplicant
+
+First, you must create a configuration file for your interface:
+
+    $ cp /etc/wpa_supplicant/wpa_supplicant.conf /etc/wpa_supplicant/wpa_supplicant-<device>.conf
+
+### WEP Configuration
+
+For WEP configuratoin, you can just add the following lines to your
+device's `wpa-supplicant.conf` :
+
+    # Example default configuration file for wpa_supplicant.conf(5).
+    ctrl_interface=/run/wpa_supplicant
+    update_config=1
+    ctrl_interface_group=wheel
+    eapol_version=1
+    ap_scan=1
+    fast_reauth=1
+
+    # Add your networks here
+    network={
+        ssid="MYSSID"
+        key_mgmt=NONE
+        wep_key0="YOUR AP WEP KEY"
+        wep_tx_keyidx=0
+        auth_alg=SHARED
+    }
+
+### WPA-PSK encryption
+
+For `WPA-PSK` encryption, we must generate a `pre shared key` with
+[wpa\_passphrase(8)](https://man.voidlinux.org/wpa_passphrase.8). To do
+so, run the following command:
+
+    wpa_passphrase <MYSSID> <key>
+
+You must append the output to your `wpa_supplicant.conf` file like so:
+
+    wpa_passphrase <MYSSID> <key> >> /etc/wpa_supplicant/wpa_supplicant-<device_name>.conf
+
+The resulting file should look something like this:
+
+    # Example default configuration file for wpa_supplicant.conf(5).
+    ctrl_interface=/run/wpa_supplicant
+    update_config=1
+    ctrl_interface_group=wheel
+    eapol_version=1
+    ap_scan=1
+    fast_reauth=1
+    # Add your networks here.
+    network={
```

Comment:
This is a duplicate of the WEP config, this should be the correct format

    network={
        ssid="MYSSID"
        #psk="YOUR AP KEY"
        psk=a8c34100ab4e5afac33cad7184d45a29ee0079001577d579bec6b74e4d7b5ac8
    }